packages/trail-fastify-server/lib/settings.js

```diff
@@ -43,7 +43,8 @@ exports.loadSettings = options => {
     },
     use: {
       restAPI: resolve('useRESTAPI', 'TRAIL_USE_REST_API', true),
-      graphQL: resolve('useGraphQL', 'TRAIL_USE_GRAPHQL', true)
+      graphQL: resolve('useGraphQL', 'TRAIL_USE_GRAPHQL', true),
+      corsPlugin: resolve('useCORS', 'TRAIL_USE_CORS', false)
```

Comment:
this setting should be the CORS origin and provided to fastify-cors origin setting, so that a user can configure the origin rather than conditionally registering the plugin. Hence, let's call this `corsOrigin` and reflect that in the name of  the setting and environment variable